libraries/entities-renderer/src/RenderableImageEntityItem.cpp

```diff
@@ -59,10 +59,22 @@ void ImageEntityRenderer::doRenderUpdateAsynchronousTyped(const TypedEntityPoint
     _alpha = entity->getAlpha();
     _pulseProperties = entity->getPulseProperties();
 
+    bool nextTextureLoaded = _texture && (_texture->isLoaded() || _texture->isFailed());
     if (!_textureIsLoaded) {
         emit requestRenderUpdate();
```

Comment:
Should the `emit requestRenderUpdate()` be done after the `if { ... }`?

Reply:
it doesn’t need to be!  the emit just tells the EntityTreeRenderer we need to update again later, we don’t necessarily need to do it last or anything